stdlib/public/core/CompilerProtocols.swift

```diff
@@ -435,6 +435,24 @@ public protocol _ExpressibleByBuiltinExtendedGraphemeClusterLiteral
     _builtinExtendedGraphemeClusterLiteral start: Builtin.RawPointer,
     utf8CodeUnitCount: Builtin.Word,
     isASCII: Builtin.Int1)
+
+  @available(SwiftStdlib 9999, *)
+  init(
+    _builtinExtendedGraphemeClusterLiteral start: Builtin.RawPointer,
+    length: Builtin.Word,
```

Comment:
`utf8CodeUnitCount` was a mouthful, but it did clarify the expected encoding of the data -- perhaps it'd be worth adding UTF-8 back in this name somewhere.